### DISCUSSION TITLE
2005/11/17/c-%E4%B8%AD%E7%9A%84%E6%8E%A5%E5%8F%A3%E4%B8%8E%E6%8A%BD%E8%B1%A1%E7%B1%BB

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#60] [Announcements]
# 2005/11/17/c-%E4%B8%AD%E7%9A%84%E6%8E%A5%E5%8F%A3%E4%B8%8E%E6%8A%BD%E8%B1%A1%E7%B1%BB

一．接口的定义

https://blog.qizhen.xyz/2005/11/17/c-%E4%B8%AD%E7%9A%84%E6%8E%A5%E5%8F%A3%E4%B8%8E%E6%8A%BD%E8%B1%A1%E7%B1%BB

**Comment by ruanqizhen** [comment]
之前博客下的留言：

Qizhen
2008年03月27日 09:14 编辑
这是我转贴的，它的例子我也没仔细看。但是它仅是用来搞清楚几个概念的，不必注意细节。

回复
Xiaobo
2008年03月26日 18:30 编辑
一个例子里面n多的错误，误人子弟啊！
